concepts/tuples/about.md

```diff
@@ -0,0 +1,119 @@
+# Tuples
+
+A [tuple][tuple] is a finite ordered list of elements which is immutable.
+Tuples requires all positions to have a fixed type this in turns means that the compiler knows what type each position is.
+The types used in a tuple can be different, but the types must be known at compile time.
+
+## Creating a Tuple
+
+Depending on if the tuples values types can be interprited under compilation, the tuple can be created in different ways.
```

Comment:
I'm not sure what you mean by this sentence. Do you mean to say that you can only create a tuple literal if the types are known at compile time?

Reply:
Yes